gitpod-remote/README.md

```diff
@@ -1,31 +1,7 @@
 # Gitpod Classic Remote
 
-> **Note**: If you’re looking for the [Gitpod Flex](https://app.gitpod.io) experience, please install the [Gitpod Flex extension](https://marketplace.visualstudio.com/items?itemName=gitpod.gitpod-flex). For [Gitpod Classic](https://gitpod.io/workspaces) workspaces, this is the extension you need.
+> **Note**: If you’re looking for the [Gitpod Flex](https://app.gitpod.io) experience, please install the [Gitpod Flex extension](https://marketplace.visualstudio.com/items?itemName=gitpod.gitpod-flex).
 
-Gitpod is an open-source Kubernetes application for automated and ready-to-code development environments that blends in your existing workflow. It enables you to describe your dev environment as code and start instant and fresh development environments for each new task directly from your browser.
+This extension is automatically installed in Gitpod Classic workspaces. It provides the Gitpod Classic experience in Visual Studio Code. It is required to provision extension, task terminals, etc.
```

Comment:
If you need Gitpod Classic workspaces, you should use the [Gitpod Classic extension](https://marketplace.visualstudio.com/items?itemName=gitpod.gitpod-desktop).

This extension is automatically installed by [Gitpod Classic extension](https://marketplace.visualstudio.com/items?itemName=gitpod.gitpod-desktop) inside workspaces in order to provision extensions, task terminals, etc. You should not install this extension manually.